src/index.js

```diff
@@ -20,11 +20,12 @@ const customBlocks = {
     restricted: false,
     mostUsed: true,
     sidebarTab: 1,
+    blockHasOwnFocusManagement: true,
     security: {
       addPermission: [],
       view: [],
     },
-    gridAllowedBlocks: ['teaser', 'image', 'listing', 'text'],
+    gridAllowedBlocks: ['teaser', 'image', 'listing', 'slate', 'text'],
```

Comment:
@tiberiuichim won't this break if slate is not present?

Reply:
https://github.com/plone/volto/blob/master/src/components/manage/BlockChooser/BlockChooser.jsx#L44

I think it would be fine. I think :)